app/services/integrations/aggregator/base_service.rb

```diff
@@ -50,6 +50,26 @@ def provider_key
         end
       end
 
+      def throttle!(*providers)
```

Comment:
this is currently not being used to check anything correct?

Reply:
It is used in the aggregator services, [here](https://github.com/getlago/lago-api/pull/2839/files#diff-150de67c3e1df586da34867a50a0b0dda236f499eceb3dec3cf2df9174f4bdf2R17) for example.